cvat/apps/consensus/new_intersect_merge.py

```diff
@@ -0,0 +1,1983 @@
+# Copyright (C) 2024 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import itertools
+import logging as log
+import math
+from collections import OrderedDict
+from typing import Any, Callable, Dict, Iterable, List, Optional, Sequence, Tuple, Type, Union, cast
+
+import attr
+import datumaro as dm
+import numpy as np
+from attr import attrib, attrs
+from datumaro.components.annotation import (
+    Annotation,
+    AnnotationType,
+    Bbox,
+    Label,
+    LabelCategories,
+    MaskCategories,
+    PointsCategories,
+)
+from datumaro.components.cli_plugin import CliPlugin
+from datumaro.components.dataset import Dataset, DatasetItemStorage, IDataset
+from datumaro.components.errors import (
+    AnnotationsTooCloseError,
+    ConflictingCategoriesError,
+    DatasetMergeError,
+    FailedAttrVotingError,
+    FailedLabelVotingError,
+    MediaTypeError,
+    MismatchingAttributesError,
+    MismatchingImageInfoError,
+    MismatchingMediaError,
+    MismatchingMediaPathError,
+    NoMatchingAnnError,
+    NoMatchingItemError,
+    VideoMergeError,
+    WrongGroupError,
+)
+from datumaro.components.extractor import CategoriesInfo, DatasetItem
+from datumaro.components.media import Image, MediaElement, MultiframeImage, PointCloud, Video
+from datumaro.util import find
+from datumaro.util.annotation_util import find_instances, max_bbox, mean_bbox
+from datumaro.util.attrs_util import ensure_cls
+from scipy.optimize import linear_sum_assignment
+
+
+def OKS(a, b, sigma=0.1, bbox=None, scale=None, visibility_a=None, visibility_b=None):
```

Comment:
Probably, it's better to just import the function from `quality_reports.py` instead of copying.